services/ui-src/src/components/modal/Modal.tsx

```diff
@@ -0,0 +1,173 @@
+import React from "react";
+
+// components
+import {
+  Box,
+  Button,
+  Heading,
+  Modal as ChakraModal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from "@chakra-ui/react";
+import { CloseIcon } from "@cmsgov/design-system";
+import { makeMediaQueryClasses } from "utils";
+
+export const Modal = ({ actionFunction, content, modalState }: Props) => {
+  const mqClasses = makeMediaQueryClasses();
+  return (
+    <ChakraModal isOpen={modalState.isOpen} onClose={modalState.onClose}>
```

Comment:
I would maybe pass the `isOpen` and `onClose` props directly rather than wrapped in `modalState` but this is fine too.